### ISSUE TITLE
A question about train loss

### ISSUE THREAD

**Issue by cslvjt** [qianqianwang68/omnimotion#35]
![image](https://github.com/qianqianwang68/omnimotion/assets/65208601/765727b7-a0fa-49fa-9a90-003d0004fa92)
When I trained on my own dataset, I found that the loss function did not converge, have you encountered this problem?

**Comment by cslvjt** [comment]
I'm using the training settings you recommended and the video has a total of 61 frames。

**Comment by cslvjt** [comment]
Another question about num_pts. What's the point of the num_pts? Thank you.

**Comment by cslvjt** [comment]
Thank you!
